### ISSUE TITLE
Apple TV3 not showing artwork in Trailers - Plex

### ISSUE THREAD

**Issue by navhawk80** [izznogooood/plexconnect-docker#6]
Hello,
Due to some changes in Plex were happened last time Apple TV3 not showing artwork in Trailers - Plex. The solution is to update python to latest virsion 2.7.18.
May I kindly ask you to update your container to let it work with appletv3 again? I think a lot of peaple still using it.
You ucan read about the issue here https://forums.plex.tv/t/apple-tv3-not-showing-artwork-in-trailers-plex/724203

**Comment by izznogooood** [comment]
Sorry for my late reply!

Ill be happy to do that!
I will release a :dev sometime after 12pm today (Sorry I have a deadline for another project) and you can test it for me? I dont have aTV anymore. ? 
 (If you are unsure how to do that please send me a screenshot of how you configure / run the container and I can lead you in the right direction)

**Comment by navhawk80** [comment]
Sure. I will test it. Take your time to another project as you wish. It is not urgent. You response is already highly appreciated. :))
I use stack in portainer to run container. Than I change default network to macvlan created via SSH in advance.
![plexconnect](https://user-images.githubusercontent.com/73139535/136598554-88dea7b9-812b-488a-a5f5-998690746f25.jpg)
) 
So some time ago all was working with this config.

One more thing I've just found is that when I try to start any movies on apple TV3 I always get "An erron occurred loading this content. Try again later". All navigation through the menu is working but without artwork ....All others devices are working even HW transcoding. I do not know if it's relaited or not but it was working...
Thanks for your effort. :)

**Comment by izznogooood** [comment]
> One more thing I've just found is that when I try to start any movies on apple TV3 I always get "An erron occurred loading this content. Try again later". 

Does this mean you cant play movies at all ?

**Comment by navhawk80** [comment]
> Does this mean you cant play movies at all ?

Correct. I can choose audio or subtitle tracks, can navigate through the menu but can't play any format of video nor HEVC nor AVC nor mpeg4.....

**Comment by izznogooood** [comment]
Can you get someone else to confirm this who is using the image? This sounds like a much bigger issue than just the python version. Maybe we can bring @moodyblue in on this ? He is maintaining plexconnect atm right ?

**Comment by navhawk80** [comment]
> This sounds like a much bigger issue than just the python version.

Looks like....

> Maybe we can bring @moodyblue in on this ?

Just found a thread with this issue with @moodyblue involved into it. It was craeated later than I created this one. So the author is aware. Will follow there. Sorry for disturbing you and thanks for your time. :)) 
https://github.com/iBaa/PlexConnect/issues/608 -thread with this issue

**Comment by izznogooood** [comment]
No problem!

Just get back to me if this turnes out to be a problem with docker-plexconnect. Remember, if it's a plexconnect problem, restarting the container after they update plexconnect will solve the issue. I'll keep this open. If you need further assistance just comment.

**Comment by moodyblue** [comment]
Thanks for the heads up. The problem affects remote access only, and is probably caused by the security changes that Plex Inc implemented on Sept/30. Not sure that I will be able to fix it.

**Comment by izznogooood** [comment]
Any news on this ?
